src/js/components/RadioButton/StyledRadioButton.js

```diff
@@ -32,7 +32,6 @@ const StyledRadioButtonInput = styled.input`
   width: 0;
   height: 0;
   margin: 0;
-  ${props => !props.disabled && 'cursor: pointer;'};
```

Comment:
what is the reason for cleaning this out?

Reply:
This line got me thinking that a 0x0 radio input (the one which is hidden would never need a cursor style as it would never receive hover. I can check this again on all browsers today.

Reply:
I checked on macOS and IE15, I don't see 0x0 checkboxes anyway.